multi-server-adding-mailbox.adoc

```diff
@@ -0,0 +1,228 @@
+[[multi-server-adding-mailbox]]
+= Adding a Mailbox Server to a Single Server Configuration
+
+In the {product-name} ({product-abbrev}) single server environment,
+the LDAP, MTA, and mailbox services are on one machine. This chapter
+explains how to add a new machine that is configured as a mailbox
+server to a single server configuration and how to remove the mailbox
+server from the single server node.
+
+== Setup Requirements For Adding a Mailbox Server
+
+* The new machine you are adding must have the same operating system,
```

Comment:
I wonder if the statement about having to have the same operating system is actually true and if there is any reason for this strict requirement (latest version, patch levels). I suspect it is possible to run Zimbra-LDAP on CentOS7, zimbra-store on Ubuntu16 and zimbra-proxy on Ubuntu14 in the same multi-server set up without any issues.